src/components/LoanCards/KivaClassicBasicLoanCard.vue

```diff
@@ -147,49 +147,51 @@
 			/>
 		</kv-ui-button>
 
-		<kv-ui-button
-			class="tw-text-secondary"
-			variant="secondary"
-			v-if="isInBasket"
-			v-kv-track-event="['Lending', 'click-Read more', 'checkout-now-button-click', loanId, loanId]"
-			to="/basket"
-		>
-			<slot>
-				<div class="tw-inline-flex tw-items-center tw-gap-1">
-					Checkout now
-					<kv-material-icon
-						class="tw-w-2.5 tw-h-2.5"
-						:icon="mdiCheckCircleOutline"
-					/>
-				</div>
-			</slot>
-		</kv-ui-button>
-
-		<!-- Lend button -->
-		<kv-ui-button
-			key="lendButton"
-			v-if="!allSharesReserved && !isLoading && showLendNowButton && !isAdding && !isInBasket"
-			class="tw-inline-flex tw-flex-1"
-			data-testid="bp-lend-cta-lend-button"
-			type="submit"
-			@click="addToBasket"
-			v-kv-track-event="[
-				'Lending',
-				'lend-button-loan-upsell',
-				expLabel
-			]"
-		>
-			{{ ctaButtonText }}
-		</kv-ui-button>
+		<!-- Template wrapper for different states of the "lend now" button -->
+		<template v-if="showLendNowButton">
+			<kv-ui-button
```

Comment:
This is the button that shows up now on the homepage when loan is in basket, I think we just need to wrap all these states